APE24.rst

```diff
@@ -0,0 +1,477 @@
+* the package maintainer is unwilling to ensure the package can be used and maintained.
+
+While in practice, rejection is rare, a rejection does *not* have to come from
+the Astropy community directly and will be objective.
+
+A package that fails the Astropy-specific criteria may still be accepted into the
+pyOpenSci ecosystem (and also published by JOSS) if it is in scope;
+an example of this might be a well-documented and developed Python package that
+is useful to the scientific community that fails to relate well to the Astropy ecosystem.
+
+"Re-review" of packages
+-----------------------
+
+**Note: This does not exist yet and probably will not happen before
+this APE is accepted. As of July 2023, pyOpenSci had a somewhat
+manual process to check in on an accepted package.**
+
+One core goal of pyOpenSci is to support scientific Python packages that are
+maintained over time. Due to the lack of resources, it does not do a *full*
+re-review of packages (involving Editor-in-Chief, editor, and 2 reviewers).
+However, it has plans to set up automated checks that track the "health"
+and maintenance level of a package over time, with the goal of identifying
+packages that have become "orphans" (i.e, unmaintained). Once a package is
+identified as needing additional maintenance, it will be flagged.
+
+"Health" might include frequency of commits, releases, CI status, etc.
+Some, if not most, of this data will be collected using
+`devstats <https://github.com/scientific-python/devstats>`_ and
+`repo-review <https://github.com/scientific-python/repo-review>`_ from
+`Scientific Python`_. This information will be displayed on public dashboards
+hosted on the pyOpenSci website.
+
+However, automation can only go so far; for instance, if automation
+is deployed in GitHub Actions, then any package hosted outside of GitHub
+would be excluded from these checks. In that case, manual intervention
+or alternate implementation might still be necessary.
+
+If the package is no longer maintained, one of the following can happen:
+
+1. If it is a widely used package and the maintainer wants to see it live on,
+   pyOpenSci will try to help the maintainer build a new maintainer team;
+2. Otherwise, pyOpenSci will gracefully sunset it from the list of maintained/accepted
+   packages.
+
+After sunsetting, if a package becomes active again, it is up to the package
+maintainer to contact Astropy or pyOpenSci in order to have it be actively
+listed once more. An example scenario that might happen is when the sole
+maintainer goes on a hiatus longer than the time frame set in
+`maintainer responsiveness <https://www.pyopensci.org/software-peer-review/our-process/policies.html#maintainer-responsiveness>`_
+and then comes back to a sunsetted package.
+
+.. _Scientific Python: https://scientific-python.org/
+
+Finding reviewers
+-----------------
+
+pyOpenSci currently has a list of reviewers who have signed up for this task using the
+`reviewer sign-up form`_.
```

Comment:
@lwasser Does pyopensci ever contact people as reviewers who have not signed-up by themselves? Knowledgeable editors will often know domain experts that might make good reviewers. Many of those might not know about pyopensci or be willing to actively opt-in, but would review a package is asked.

Reply:
@hamogu absolutely. in fact often that is just what happens by default because editors as you point out know people. that is why it's really helpful to have an editor with domain expertise who knows peopel in the community! 

the other thing that happens is a reviewer signs up and when they sign up they have time. Then  we need their expertise say 3-5 months later and they are busy. so the signup / list of people is useful but sometimes doesn't represent who has time when we need that specific expertise.